### PR TITLE
Gutlunches now spawn on lavaland.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## What Does This PR Do
Gutlunches (and more rarely gubbucks) can now spawn on lavaland.
They are a non-hostile fauna, previously found in ash walker den.

Spawn chance is currently at four, I'd prefer to see it in game first to (possibly) adjust it.
There is also 5% (more than regular "rare" mobs) chance for it to become a gubbuck - they effectively are the same, just smaller and colorful.

## Why It's Good For The Game
More diversity for Lavaland fauna - having something that only _HOPES_ you die is pretty sweet after going through hordes of stuff that _ACTUALLY_ kills you!
This also makes sense lore wise on many levels!

1. We now know how did ashies get theirs.
2. It makes sense for scavenger fauna to develop on a murder planet.
3. It makes sense for non-violent fauna to develop on any planet.

okay that was only three... anyways!

## Images of changes
obraz

## Testing
observed, saw a couple on observer menu. Re-generated LL a couple times to get data for spawn chance - it's a long and inconsistent process however.

## Changelog
:cl:
tweak: Gutlunches can now be naturally found in lavaland wastes.
/:cl:

### CODE REVIEW

**Comment by Venuska1117** [comment]
what if you get gibbed by megafauna, and this dude will be nearby?
will it make you perma-dead? (they eat gibs, even brains)

**Comment by meow20** [comment]
> what if you get gibbed by megafauna, and this dude will be nearby? will it make you perma-dead? (they eat gibs, even brains)

then don't play miner if getting permakilled is an issue :D

**Comment by Sadhorizon** [comment]
Them not targeting brains would be a cool buff, yes.

Reason? Not yummy enough!

**Comment by ATP-Engineer** [comment]
Can you make it so most if not all other lavaland mobs don't target them because of how much they'd probably eat shit and die to random fauna before finally being able to eat random miner that got gibbed by dragon?

**Comment by Sadhorizon** [comment]
> Can you make it so most if not all other lavaland mobs don't target them because of how much they'd probably eat shit and die to random fauna before finally being able to eat random miner that got gibbed by dragon?

Heyo! They actually already use the same faction as other fauna!

**Comment by Inferno1882** [comment]
I could be wrong in this statement, so do correct me. I make no claim of intelligence on my part.

On the subject of "getting deleted via brain consumption", Megafauna in their current state do the following to corpses:

Vetus Speculator - Dusts, Instant Removal
Blood-Drunk - Gibs
Bubblegum - From what I'm seeing it doesn't gib/dust in any form, but having never died to one/remember dying to one, I can't be certain.
Colossus - Dusts, Instant Removal
Drake - Gibs
Heirophant - Dusts, Instant Removal
Legion (the Big One) - Codewise, it doesn't look like it calls a gib or dust either, but, again, as I have not bothered fighting one before, I cannot be certain.

From this list, 3 of the 7 Megafauna, upon beating a player into a pulp, instantly removes them from round (in original form) via the `L.dust()` action. Only two, being the Drake and BDM, will actually gib (with absolute certainty) via L.gib(), and the last two... well, I defer to anyone who actually knows.

With this being said, the chances that you as a player are both gibbed and near a gutlunch/gubbuck to see your grey matter slurped into their little mouths would be low. I, personally, don't mind letting them do that, because it also gives a Lavaland-based antagonist (who really doesnt like you, for some reason) just one more way to dispose of a brain, that isn't just chasm/lava/blending it. Then again, that's not the point of this PR. 

tl;dr good PR, nice to see them getting some love.

**Review by lewcc — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]
